apps/website/get-started/updating/README.md

```diff
@@ -15,9 +15,3 @@ If you are using either the React or Angular packages, you should update them at
 npm install @cds/core@latest @cds/angular@latest # updates both Core and Angular
 npm install @cds/core@latest @cds/react@latest # updates both Core and React
 ```
-
-If there is a prerelease version available for testing, you can install it using the `next` tag on NPM. These versions are only available when we are actively working on a new major version, so only do this if you are willing to try beta software.
```

Comment:
Rather than removing this can we add some clarity around the next tag? E.g - that is only available as new releases are prepared for release. 

IMO - folks should always upgrade to the latest, not next.